distribution/docs/src/main/resources/content/_core-concepts/data-standards.adoc

```diff
@@ -0,0 +1,39 @@
+:type: subCoreConcept
+:status: published
+:title: Data Standards added by ${cal-branding}
+:parent: Standards Supported by ${branding}
+:order: 011
+
+.{title}
+[cols="2,1m,1" options="header"]
+|===
+
+|Format
+|File Extensions
+|Additional Metadata Attributes Available (if populated)
+
+|http://www.gwg.nga.mil/ntb/baseline/docs/2500c/index.html[NITF 2.0 and 2.1]
+|ntf
+|<<_file_formats_supported,Standard attributes>> and<<_nitf_attribute_mappings,NITF Attribute Mappings>>
+
+|http://www.gwg.nga.mil/ntb/baseline/docs/NSIF/[NSIF/STANAG 4545]
+|ntf
+|<<_file_formats_supported,Standard attributes>> and<<_nitf_attribute_mappings,NITF Attribute Mappings>>
+
+|http://www.gwg.nga.mil/misb/docs/nato_docs/STANAG_4609_Ed3.pdf[STANAG 4609 FMV]
+|fmv
+<<_file_formats_supported,Standard attributes>>
```

Comment:
Missing a | here, causes the table to render incorrectly.

Reply:
fixed